src/utils/local_stats_alloc/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
-name = "stats_alloc"
+name = "local_stats_alloc"
 version = "0.1.11"
 edition = "2021"
-description = "Allocator with statistics"
+description = "Local allocator with statistics"
```

Comment:
What does the "local" means here?

Reply:
It is used as an allocator in `new_in` function. Currently, we used it to estimate the hash join cache size. 

The global is used as an Global allocator.